### PR TITLE
Restyled error page

### LINKED ISSUE #616: Image Positioning on Activities page
**Describe the bug**
On the mobile screen, the “no activities created” page and image is positioned under the header.

**To Reproduce**

1. Open the mobile screen
2. Click on Browse Activities
3. See error

**Expected behavior**
There should be a margin top to show the image clearer

**Screenshots**
D98D6239-7DBC-4ECE-9613-ED7550C601BF

### PR DESCRIPTION
**Pull request by Vic-Orlands**
## Summary

Description of PR here...
On the mobile screen, the “no activities created” page and image is positioned under the header.

Closes #616 

## Changes
Added margins and correctly placed the error page below the header

## Screenshots
Screenshot from 2023-03-16 03-05-07

### CODE REVIEW

File: zubhub_frontend/zubhub/package-lock.json
Comment (tuxology) [inline zubhub_frontend/zubhub/package-lock.json:1]: Please remove this file. We don't want to overwrite the current lock file.

File: zubhub_frontend/zubhub/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "react-quill": "^2.0.0",
     "react-redux": "^7.2.2",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^5.0.1",
```
Comment (tuxology) [inline zubhub_frontend/zubhub/package.json]: Is this really needed? We don't want to make major changes in versions for now

**Review by tuxology — CHANGES_REQUESTED** [review]

File: zubhub_frontend/zubhub/package-lock.json
Comment (Vic-Orlands) [reply zubhub_frontend/zubhub/package-lock.json:1]: Alright, Let me do that now

File: zubhub_frontend/zubhub/package.json (hunk 1 above)
Comment (Vic-Orlands) [reply zubhub_frontend/zubhub/package.json]: Okay, I completely understand

Comment (Vic-Orlands) [reply zubhub_frontend/zubhub/package.json:35]: Okay, I completely understand

Comment (Vic-Orlands) [reply zubhub_frontend/zubhub/package.json:35]: Okay, I completely understand

**Comment by Vic-Orlands** [comment]
@tuxology I have resolved the issues

**Review by tuxology — APPROVED** [review]

**Pull request merged**